### PR TITLE
Use sudo for apt commands

### LINKED ISSUE #193: Github some checks were not successful
github workflow: pyrsia/.github/workflows/demo.yml report error. Details as the below: 
Run apt-get update
Reading package lists...
E: Could not open lock file /var/lib/apt/lists/lock - open (13: Permission denied)
E: Unable to lock directory /var/lib/apt/lists/
W: Problem unlinking the file /var/cache/apt/pkgcache.bin - RemoveCaches (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/srcpkgcache.bin - RemoveCaches (13: Permission denied)
Error: Process completed with exit code 100.

### PR DESCRIPTION
**Pull request by sbtaylor15**
Resolves #193

### CODE REVIEW

File: .github/workflows/demo.yml [hunk 1]
```diff
@@ -8,15 +8,16 @@ jobs:
...
-          apt-get install -y pyrsia
-        shell: bash 
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y wget gnupg
+          wget -qO - https://pyrsia.github.io/public.key | sudo apt-key add -
+          echo "deb https://pyrsia.github.io/repo bionic main" | sudo tee -a /etc/apt/sources.list > /dev/null
+          sudo apt-get update
+          sudo apt-get install -y pyrsia
+          sudo chmod 755 /usr/local/bin/pyrsia
```
Comment (prince-chrismc) [inline .github/workflows/demo.yml:18]: I would expect the installer to have done this 🤔 separate issue?

**Comment by prince-chrismc** [comment]
Also can we sneak in using the new `pyrsia.io` address?

File: .github/workflows/demo.yml (hunk 1 above)
Comment (sbtaylor15) [reply .github/workflows/demo.yml]: Changes permissions at the package level so the chmod is no longer needed.

File: .github/workflows/demo.yml [hunk 2]
```diff
@@ -8,15 +8,15 @@ jobs:
...
-          wget -qO - https://pyrsia.github.io/public.key | apt-key add -
-          echo "deb https://pyrsia.github.io/repo bionic main" >> /etc/apt/sources.list
-          apt-get update
-          apt-get install -y pyrsia
-        shell: bash 
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y wget gnupg
+          wget -qO - https://pyrsia.github.io/public.key | sudo apt-key add -
+          echo "deb https://pyrsia.github.io/repo bionic main" | sudo tee -a /etc/apt/sources.list > /dev/null
```
Comment (prince-chrismc) [inline .github/workflows/demo.yml:15]: Can we be brave? 🙈 

```suggestion
          wget -qO - https://pyrsia.io/public.key | sudo apt-key add -
          echo "deb https://pyrsia.io/repo bionic main" | sudo tee -a /etc/apt/sources.list > /dev/null
```

Comment (sbtaylor15) [reply .github/workflows/demo.yml:15]: Missed that.  Let me change it.

**Review by fishseabowl — APPROVED** [review]
It looks good for me.

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**